.travis.yml

```diff
@@ -26,6 +26,8 @@ env:
   - TESTS="1" ODOO_REPO="OCA/OCB"
   - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="website_signup_legal_page_required"
   - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="website_signup_legal_page_required"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_seo_redirection"
```

Comment:
This is on purpose. From now on, this module will intercept all HTTP requests of Odoo when installed, and Travis installs all modules. It is supposed to work fine, but if it does not, you could become crazy to find the problem. It's there to help locating the problem in case it fails, assuming my code is not perfect.